Tiltfile

```diff
@@ -103,7 +103,6 @@ else:
             "ratelimit",
             "redis",
         ],
-        port_forwards=["3000:3000"],
```

Comment:
PATH running with Envoy should not expose port 300 as all traffic should go through Envoy on port 3001.